ksqldb-engine/src/main/java/io/confluent/ksql/physical/pull/HARouting.java

```diff
@@ -200,8 +206,11 @@ private void executeRounds(
       try {
         final PartitionFetchResult fetchResult = future.get();
         if (fetchResult.isError()) {
+          exceptionsPerPartition.computeIfAbsent(
```

Comment:
The exceptions thrown by standby nodes are added as suppressed exceptions. Previously, they were lost